### DISCUSSION TITLE
Έργο ανοιχτού λογισμικού | Π2016072

### DISCUSSION THREAD

**Discussion by AlxiKan** [courses-ionio/cscw#173] [Q&A] [answered]
Μπορώ να συνεισφέρω στην κατασκευή ιστοσελίδας του τμήματος;

https://github.com/ioniodi/sitegr/

**Comment by epidrome** [comment, marked as answer]
@AlxiKan ναι, υποχρεωτικά σε αυτό το μάθημα, αφού είναι στο πρόγραμμα. 

η σχετική ανακοίνωση αφορά τα άλλα έργα και κυρίως μια δική σας επιλογή #171
